README.md

```diff
@@ -48,11 +48,19 @@ var childWalletSecret = lib.derivePublic(parentWalletPublicKey, 1, 1)
 * `Buffer toPublic(Buffer privateKey)`
 * `Buffer decodePaperWalletMnemonic(string paperWalletMnemonic)`
 * `Buffer xpubToHdPassphrase(Buffer xpub)`
-* `string packAddress(Array[int] derivationPath, Buffer xpub, Buffer hdPassphrase, int derivationScheme)`
+* `Buffer packBootstrapAddress(Array[int] derivationPath, Buffer xpub, Buffer hdPassphrase, int derivationScheme)`
+* `Buffer packBaseAddress(Buffer pubKey, Buffer stakePubKey, int addressType, int networkId, Bool isStakeHash = false)`
+* `Buffer packPointerAddress(Buffer pubKey, Object pointer, int addressType, int networkId)`
+* `Buffer packEnterpriseAddress(Buffer pubKey, int addressType, int networkId)`
```

Comment:
note for refactor -> addressType parameters appears redundant given that this is determined by the function itself, so we can remove it